Commands/Play.spec.js

```diff
@@ -0,0 +1,30 @@
+import play from "./Play";
+
+describe("Tests the play function", () => {
```

Comment:
You can think of your test descriptions as a sentence without the keywords. For example, as currenntly written, this would say: 

"Tests the play function will error if you are not in a voice channel" 
"Test the play function will join channel to play mp3 if command is available and user is in room" 

Which sounds better if you marked it up like this

```js
describe("The Play Function", () => {
  it("throws an error if you are not in a voice channel", async () => {
  it("joins a channel to play mp3 if the command is available and user is in room", async () => {

Reply:
As in, "The Play Function throws an error if you are not in a voice channel." sounds like a sentence.

Reply:
Noted, I will change this. Thank you.